dotcom-rendering/index.d.ts

```diff
@@ -551,6 +551,22 @@ interface TagType {
 	bylineImageUrl?: string;
 }
 
+interface BlocksRequest {
```

Comment:
Where does this come from? Could we have some JSDoc info as part of the interface?

```ts
/**
 * A request from [frontend][]. This type is kept in sync with Scala’s manually.
 *
 * [frontend]: https://github.com/guardian/frontend/blob/…
 */
interface BlocksRequest {
```

Reply:
Comment added 👍 

In general though, this is a very long standing issue where we have no way to share types between Frontend and DCR. Personally, I think GraphQL would be a good fit here but that's a _much_ larger discussion than this PR!